apps/prairielearn/elements/pl-matching/pl-matching.py

```diff
@@ -355,16 +370,8 @@ def render(element_html: str, data: pl.QuestionData) -> str:
         }
 
         if score is not None:
-            try:
-                score = float(score)
-                if score >= 1:
-                    html_params["correct"] = True
-                elif score > 0:
-                    html_params["partial"] = math.floor(score * 100)
-                else:
-                    html_params["incorrect"] = True
-            except Exception as exc:
-                raise ValueError("invalid score" + score) from exc
+            score_type, score_value = pl.determine_score_params(score)
+            html_params[score_type] = score_value
```

Comment:
Made this change because the pyright threw an error on the code block that was here before.